kzz7yh-f33279/cod-ve09v2_kzz7yh-f33279.xml

```diff
@@ -137,8 +138,8 @@
       <lb ed="#V" n="98"/>aliqua species. Ergo materiam dic quandam informitatem sne omni actualtatem 
    </p>
                <p xml:id="kzz7yh-f33279-d1e229">
-                  <lb ed="#V" n="99"/>Contra Auicen. 2. meta. ca. veltio dicit quod forma 
-      priori<lb ed="#V" n="100" break="no"/>Hyle id est materiam. Sed actus non est prior possibili 
+                  <lb ed="#V" n="99"/>Contra Auicen. 2. meta. ca. veltio dicit quod forma prior est
```

Comment:
@RobbieBuhite @evyryan this is tricky abbreviation, but it is common so I would review it.
```suggestion
                  <lb ed="#V" n="99"/>Contra Auicen. 2. meta. ca. vltimo dicit quod forma prior est
```